tests/test_environment_BondOrder.py

```diff
@@ -113,7 +113,7 @@ def test_points_ne_query_points(self):
         r_max = 1.6
 
         num_neighbors = 12
-        n_bins_theta = 30
+        n_bins_theta = 35
```

Comment:
This seems unrelated.

Reply:
It is related, surprisingly. The changes in this PR have, for whatever reason, created a situation where on Windows only, choosing exactly 30 bins created a situation where a small fraction of the bin counts leaked over into neighboring bins and made the test fail.

Reply:
Ah, well that is fine then. Numerical precision and all that.